sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -213,6 +220,12 @@ Licensed under the MIT License.
       <version>3.9.0</version> <!-- {x-version-update;org.mockito:mockito-core;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-exporter-logging</artifactId>
```

Comment:
This is temporary change due to bug in event api #22115 . Will revert before merging the PR